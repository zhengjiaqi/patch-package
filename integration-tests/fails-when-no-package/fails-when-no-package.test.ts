import { runIntegrationTest } from "../runIntegrationTest"
runIntegrationTest({
  projectName: "fails-when-no-package",
  shouldProduceSnapshots: true,
})
