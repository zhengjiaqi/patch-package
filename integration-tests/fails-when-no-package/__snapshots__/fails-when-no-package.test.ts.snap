// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test fails-when-no-package: no package present failure 1`] = `
"SNAPSHOT: no package present failure
Error: Patch file found for package left-pad which is not present at node_modules/left-pad
---
patch-package finished with 1 error(s).
END SNAPSHOT"
`;
